main/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bytebase/dbcost/client"
+	"github.com/bytebase/dbcost/client/aws"
+	"github.com/bytebase/dbcost/client/gcp"
+	"github.com/bytebase/dbcost/store"
+)
+
+var cloudProvider = make(map[store.CloudProvider]client.Client)
+
+func init() {
+	APIKeyGCP := os.Args[1]
```

Comment:
It was designed by purpose. I use the env variable in starting script `start.sh`, rather than here. Because the user may want to fetch their own data, and they have no idea of this variable. They can easily do so by passing in their own API key as a parameter.

Reply:
I think we can return an error from main() if that environment key isn't set. Right now, the flag exists in both environment (render) and argument (main.go argument). We may want to pick one or the other.